mon-pix/app/templates/authenticated/user-account.hbs

```diff
@@ -27,7 +27,7 @@
             {{#if this.displayLanguageSwitch}}
```

Comment:
Suggestion : s'aligner sur ce qu'on a fait dans le ticket https://github.com/1024pix/pix/pull/6026/files ?

Reply:
:+1: Il faut effectivement chercher à ce que les 2 PR soient le plus synchronisées possibles. Ceci dit je ne vois pas ce qu'on peut faire de plus maintenant, particulièrement il ne faut pas utiliser de `aria-label` car le composant `PixSelect`utilise, lui, label avec donc l'attribut `label`.

Reply:
Je pensais à utiliser le nouveau getter `isFranceDomain` par exemple au lieu de `this.url.isFrenchDomainExtension` https://github.com/1024pix/pix/pull/6026/commits/02ba2ee84389ef6d8f62d9abeb3dba83cb2a908f par exemple